code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -127,6 +127,10 @@ Borg Hypospray
 	if(empty)
 		usr << "<span class='warning'>It is currently empty! Allow some time for the internal syntheszier to produce more.</span>"
 
+/obj/item/weapon/reagent_containers/borghypo/hacked
+	icon_state = "borghypo_s"
+	reagent_ids = list ("facid", "mutetoxin", "cyanide", "sodium_thiopental", "heparin", "teslium")

```

Comment:
What do these chemicals do when injected into someone? Mutetoxin mutes them, but the others?
(And does injecting fluorosulfuric acid into someone melt their stuff?)